process/css-styleguide/default.css

```diff
@@ -0,0 +1,105 @@
+@import url("https://fonts.googleapis.com/css2?family=Poppins:wght@300;400;600&display=swap");

```

Comment:
Poppins should be used for headings and Inter for the rest. Inter should also be imported in this file